src/backend/decorators/action/action-decorator.ts

```diff
@@ -281,10 +281,7 @@ class ActionDecorator {
       return true
     }
 
-    throw new ForbiddenError(this._admin.translateMessage('forbiddenError', resource.id(), {
-      actionName: this.name,
-      resourceId: resource.id(),
-    }))
+    throw new ForbiddenError('forbiddenError')
```

Comment:
you removed `actionName` and `resourceId`

Reply:
translations are done on frontend

Reply:
yes, but does frontend have `actionName` and `resourceId`?

Reply:
I hope, but nice to have example to assure that